cmd/immuclient/audit/auditor.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2019-2020 vChain, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package audit
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/codenotary/immudb/cmd/immuclient/service"
+	"github.com/codenotary/immudb/pkg/client"
+	"github.com/codenotary/immudb/pkg/client/auditor"
+	"github.com/codenotary/immudb/pkg/client/cache"
+	"github.com/spf13/viper"
+)
+
+const (
+	name        = "immuclient"
+	description = "immuclient"
+)
+
+var ErrAgentNotActive = errors.New("agent not active")
+
+func (cAgent *auditAgent) InitAgent() (AuditAgent, error) {
+	srv, err := service.NewDaemon(name, description, name)
+	if err != nil {
+		return nil, err
+	}
+	ctx := context.Background()
+	cAgent.Daemon = srv
+	freqstr := os.Getenv("trust-check-interval")
```

Comment:
@volknanebo is this env var name ok? i thought the trust checker name is obsolete and we use audit(or) instead.